rotkehlchen/db/filtering.py

```diff
@@ -295,6 +311,9 @@ def make(
             from_ts: Optional[Timestamp] = None,
             to_ts: Optional[Timestamp] = None,
             tx_hash: Optional[EVMTxHash] = None,
+            filter_by_events: Optional[bool] = None,  # If True, filters by events data
```

Comment:
Two things here.

Why give this argument? Why not just filter by events data if either the `protocol` or the `asset`argument is not `None`?

Reply:
Yes, could be removed